java/debugger.jpda/src/org/netbeans/modules/debugger/jpda/models/ShortenedStrings.java

```diff
@@ -130,7 +213,14 @@ static String getStringWithLengthControl(StringReference sr) throws InternalExce
         try {
             ReferenceType st = ObjectReferenceWrapper.referenceType(sr);
             ArrayReference sa = null;
+            //only applicable if the string implementation uses a byte[] instead
+            //of a char[]
+            boolean isUTF16 = false;
+            boolean isCompactImpl = false;
```

Comment:
Please add a comment like:
> JEP 254: Compact Strings after the boolean
that might help people to later understand the intention